### PR TITLE
[pt-BR] Updated translations

### PR DESCRIPTION
**Pull request by xgrind**

### CODE REVIEW

**Comment by xgrind** [comment]
Translated files must have 'declare(strict_types=1);' ?

**Comment by kenjis** [comment]
@paulbalandan 
What is this failure? It seems there is no "Migrations.0" in `Migrations.php`.
```
There was 1 failure:

1) Translations\Tests\BrazilianTranslationTest::testAllIncludedLanguageKeysAreConfigured with data set "pt-BR" ('pt-BR')
Failed asserting that the translated language key "Migrations.0" in "pt-BR" locale is configured in the main repository.
Failed asserting that an array is empty.

/home/runner/work/translations/translations/tests/Language/AbstractTranslationTestCase.php:206
```
https://github.com/codeigniter4/translations/actions/runs/8776153589/job/24080540915?pr=446

**Comment by kenjis** [comment]
> Translated files must have 'declare(strict_types=1);' ?

No.

File: Language/pt-BR/Migrations.php [hunk 1]
```diff
@@ -12,47 +12,47 @@
...
+    'migrated'          => 'Migrations concluídas.', // 'Migrations complete.',
+    'migInvalidVersion' => 'Número da versão fornecida é inválido.', // 'Invalid version number provided.',
+    'toVersionPH'       => 'Migrando para a versão %s...', // 'Migrating to version %s...',
+    'toVersion'         => 'Migrando para a versão atual...', // 'Migrating to current version...',
+    'rollingBack'       => 'Revertendo todas as migrações...', // 'Rolling back migrations to batch: ',
+    'noneFound'         => 'Nenhuma migration foi encontrada.', // 'No migrations were found.',
+    'migSeeder'         => 'Nome do seeder', // 'Seeder name',
+    'migMissingSeeder'  => 'Você deve fornecer o nome de um seeder.', // 'You must provide a seeder name.',
+    'nameSeeder'        => 'Nomeie o arquivo seeder', // 'Name the seeder file',
+    'removed'           => 'Revertendo: ', 'Rolling back: ',
```
Comment (kenjis) [inline Language/pt-BR/Migrations.php:48]:
```suggestion
    'removed'           => 'Revertendo: ', // 'Rolling back: ',
```

**Comment by kenjis** [comment]
Please fix the coding style:
```diff
  1) Language/pt-BR/Number.php (no_trailing_whitespace)
      ---------- begin diff ----------
--- /home/runner/work/translations/translations/Language/pt-BR/Number.php
+++ /home/runner/work/translations/translations/Language/pt-BR/Number.php
@@ -11,14 +11,14 @@
 
 // Number language settings
 return [
-    'terabyteAbbr' => 'TB', 
-    'gigabyteAbbr' => 'GB', 
-    'megabyteAbbr' => 'MB', 
-    'kilobyteAbbr' => 'KB', 
+    'terabyteAbbr' => 'TB',
+    'gigabyteAbbr' => 'GB',
+    'megabyteAbbr' => 'MB',
+    'kilobyteAbbr' => 'KB',
     'bytes'        => 'Bytes',
 
     // don't forget the space in front of these!
-    'thousand'    => ' mil', // ' thousand', 
+    'thousand'    => ' mil', // ' thousand',
     'million'     => ' milhão', // ' million',
     'billion'     => ' bilhão', // ' billion',
     'trillion'    => ' trilhão', // ' trillion',

      ----------- end diff -----------

   2) Language/pt-BR/Validation.php (no_trailing_whitespace)
      ---------- begin diff ----------
--- /home/runner/work/translations/translations/Language/pt-BR/Validation.php
+++ /home/runner/work/translations/translations/Language/pt-BR/Validation.php
@@ -69,7 +69,7 @@
     'uploaded' => '{field} não é um arquivo de upload válido.', // '{field} is not a valid uploaded file.',
     'max_size' => '{field} é um arquivo muito grande.', // '{field} is too large of a file.',
     'is_image' => '{field} não é um arquivo de imagem válida do upload.', // '{field} is not a valid, uploaded image file.'
     'invalidAddress'       => 'Endereço de e-mail inválido: "{0}"', // Invalid email address: "{0}"',
     'attachmentMissing'    => 'Não foi possível localizar o seguinte anexo de e-mail: "{0}"', // 'Unable to locate the following email attachment: "{0}"'
-    'attachmentUnreadable' => 'Não foi possível abrir este anexo: "{0}"', // 'Unable to open this attachment: "{0}"', 
+    'attachmentUnreadable' => 'Não foi possível abrir este anexo: "{0}"', // 'Unable to open this attachment: "{0}"',
     'noFrom'               => 'Não é possível enviar e-mail sem o cabeçalho "From".', // 'Cannot send mail with no "From" header.',
     'noRecipients'         => 'Você deve incluir destinatários: To, Cc, ou Bcc', // 'You must include recipients: To, Cc, or Bcc',
     'sendFailurePHPMail'   => 'Não foi possível enviar e-mail usando PHP mail(). Seu servidor pode não estar configurado para enviar e-mail usando este método.', // 'Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.',

      ----------- end diff -----------

   6) Language/pt-BR/Cache.php (single_line_comment_spacing)
      ---------- begin diff ----------
--- /home/runner/work/translations/translations/Language/pt-BR/Cache.php
+++ /home/runner/work/translations/translations/Language/pt-BR/Cache.php
@@ -11,8 +11,8 @@
 
 // Cache language settings
 return [
-    'unableToWrite'   => 'O cache está incapaz de escrever em "{0}"', //'Cache unable to write to "{0}".',
-    'invalidHandlers' => 'A configuração de cache deve ter uma array de $validHandlers.', //'Cache config must have an array of $validHandlers.',
-    'noBackup'        => 'A configuração de cache deve ter um handler e um backupHandler definido.', //'Cache config must have a handler and backupHandler set.',
-    'handlerNotFound' => 'A configuração de cache possui um handler ou backup handler inválido na definição.', //'Cache config has an invalid handler or backup handler specified.',
+    'unableToWrite'   => 'O cache está incapaz de escrever em "{0}"', // 'Cache unable to write to "{0}".',
+    'invalidHandlers' => 'A configuração de cache deve ter uma array de $validHandlers.', // 'Cache config must have an array of $validHandlers.',
+    'noBackup'        => 'A configuração de cache deve ter um handler e um backupHandler definido.', // 'Cache config must have a handler and backupHandler set.',
+    'handlerNotFound' => 'A configuração de cache possui um handler ou backup handler inválido na definição.', // 'Cache config has an invalid handler or backup handler specified.',
 ];

      ----------- end diff -----------


Found 6 of 988 files that can be fixed in 4.303 seconds, 28.000 MB memory used
```
https://github.com/codeigniter4/translations/actions/runs/8776836692/job/24080928693?pr=446

**Comment by xgrind** [comment]
michael@michael-desktop:~/projetos/translations$ ./bin/test pt-BR
PHP Warning:  require(/home/michael/projetos/translations/bin/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php): Failed to open stream: No such file or directory in /home/michael/projetos/translations/bin/test on line 15
PHP Fatal error:  Uncaught Error: Failed opening required '/home/michael/projetos/translations/bin/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php' (include_path='.:/usr/share/php') in /home/michael/projetos/translations/bin/test:15
Stack trace:
#0 {main}
  thrown in /home/michael/projetos/translations/bin/test on line 15
michael@michael-desktop:~/projetos/translations$

**Review by kenjis — APPROVED** [review]

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**